ui/package-lock.json

```diff
@@ -77,6 +78,7 @@
         "@perses-dev/panels-plugin": "0.41.1",
         "@perses-dev/plugin-system": "0.41.1",
         "@perses-dev/prometheus-plugin": "0.41.1",
+        "@perses-dev/tempo-plugin": "0.1.0",
```

Comment:
looks like the `package-lock.json` is indeed not aligned with `package.json`.

Perhaps a simple `npm install` could fix it !

Reply:
Yeah, I had a suspicion it might be that. I should have included this in the previous push. Okay, I'm going to update package-lock.json and push again.